scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-magnum-template.yaml

```diff
@@ -20,7 +20,9 @@
             cloudsource=develcloud{version}
             nodenumber=3
             nodenumbercompute=2
+            want_node_os=suse-12.2=3
             want_magnum=1
+            want_ceph=0
```

Comment:
I think `want_ceph=0`  is enough.  want_node_os would actually be needed when you want some nodes with SP1 and some with SP2